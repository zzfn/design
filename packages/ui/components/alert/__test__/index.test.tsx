import renderer from 'react-test-renderer';
import Alert from '../alert';

it('renders correctly', () => {
  const component = renderer.create(<Alert>Facebook</Alert>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  // re-rendering
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
