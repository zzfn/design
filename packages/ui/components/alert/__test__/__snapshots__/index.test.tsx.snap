// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="zzf-alert zzf-alert-info"
>
  Facebook
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="zzf-alert zzf-alert-info"
>
  Facebook
</div>
`;
